packages/strapi-admin/index.js

```diff
@@ -152,7 +176,48 @@ async function build({ dir, env, options }) {
   });
 }
 
+async function watch({ dir, port, options }) {
+  const entry = path.join(dir, '.cache', 'admin', 'src', 'app.js');
+  const dest = path.join(dir, 'build');
+  const env = 'development';
+
```

Comment:
this function should create the cache dir first then start the dev server then start the watcher